day_02.md

```diff
@@ -38,59 +38,60 @@
 >
 > **The syntax of a language defines its surface form.** Text-based computer languages are based on sequences of characters, while visual programming languages are based on the spatial layout and connections between symbols (which may be textual or graphical). Documents that are syntactically invalid are said to have a syntax error.
 >
-> Syntax therefore refers to the form, and it is contrasted with [semantics](https://en.wikipedia.org/wiki/Semantics_(computer_science)) – the meaning. In processing computer languages, semantic processing generally comes after syntactic processing; however, in some cases, semantic processing is necessary for complete syntactic analysis, and these are done together or concurrently.
+> Syntax therefore refers to the form, and it is contrasted with [semantics](<https://en.wikipedia.org/wiki/Semantics_(computer_science)>) – the meaning. In processing computer languages, semantic processing generally comes after syntactic processing; however, in some cases, semantic processing is necessary for complete syntactic analysis, and these are done together or concurrently.
 >
-> Computer language syntax is generally distinguished into three levels:
+> Computer language syntax is generally separated into three levels:
 >
 > - **Words** – the **lexical level**, determining how **characters** form **tokens**;
 > - **Phrases** – the **grammar level**, narrowly speaking, determining how **tokens** form **phrases**;
 > - **Context** – determining what objects or variables names refer to, if types are valid, etc.
 >
-> Distinguishing in this way yields modularity, allowing each level to be described and processed separately and often independently. First, a lexer turns the linear sequence of characters into a linear sequence of [tokens](https://en.wikipedia.org/wiki/Token_(parser)); this is known as "lexical analysis" or "[lexing](https://en.wikipedia.org/wiki/Lexical_analysis)". Second, the parser turns the linear sequence of tokens into a hierarchical syntax tree; this is known as "[parsing](https://en.wikipedia.org/wiki/Parsing)" narrowly speaking. Thirdly, the contextual analysis resolves names and checks types.
+> Distinguishing in this way yields modularity, allowing each level to be described and processed separately and often independently. First, a lexer turns the linear sequence of characters into a linear sequence of [tokens](<https://en.wikipedia.org/wiki/Token_(parser)>); this is known as "lexical analysis" or "[lexing](https://en.wikipedia.org/wiki/Lexical_analysis)". Second, the parser turns the linear sequence of tokens into a hierarchical syntax tree; this is known as "[parsing](https://en.wikipedia.org/wiki/Parsing)" narrowly speaking. Thirdly, the contextual analysis resolves names and checks types.
```

Comment:
Please check surrounding < and > for URLs inside markdown hyperlink notation. (was it suggested by a linter?)
Verify this everywhere it's been added.